.github/workflows/format.yml

```diff
@@ -33,9 +33,19 @@ jobs:
         - name: Run Black
           run: black ./ --config pyproject.toml
```

Comment:
I don't think this section is required. The concern is if we format the code before checking it, then the workflow will always pass.